db.dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM postgres:15
```

Comment:
there's already such a dockerfile in `docker/database/Dockerfile` -- can we just use that and not add this?

Reply:
feel free to overwrite

Reply:
https://github.com/flashbots/builder-hub/blob/main/docker/database/Dockerfile